spec/models/active_shipping_calculator_spec.rb

```diff
@@ -108,11 +106,11 @@ module ActiveShipping
 
     describe "compute" do
       it "should use the carrier supplied in the initializer" do
-        stub_request(:get, /http:\/\/production.shippingapis.com\/ShippingAPI.dll.*/).
-          to_return(:body => fixture(:normal_rates_request))
-        calculator.compute(package).should == 14.1
+        carrier.should_receive(:find_rates).and_return(ActiveShipping::RateResponse.new(true, "Foo"))
```

Comment:
On second thought, the `BogusCarrier` should already return a fake `RateResponse` so you might not even need to stub this here. See https://github.com/solidusio-contrib/solidus_active_shipping/blob/master/spec/lib/spree/active_shipping/bogus_carrier.rb#L11